### PR TITLE
WH-108: Teams and universities when not logged in

### PR DESCRIPTION
**Pull request by vsoler27**
SLAYED THIS PAGE IS FINALLY DONE !!!

### CODE REVIEW

File: src/pages/teamsanduniversities/teamsanduniversities/teamsanduniversities.js [hunk 1]
```diff
@@ -16,67 +16,88 @@ const TeamsAndUniversities = () => {
...
-                let myHeaders = new Headers();
-                myHeaders.append("Content-Type", "application/json");
-                myHeaders.append("x-access-token", user.token);
-    
+                changeToken(user.token); 
+            } else {
+                const raw = JSON.stringify({
+                    "email": "testUser@rit.edu",
+                    "password": "Password1",
+                });
```
Comment (abp6318) [inline src/pages/teamsanduniversities/teamsanduniversities/teamsanduniversities.js:36]: So if the user isn't logged in, we're hard coding them an account?

Comment (abp6318) [reply src/pages/teamsanduniversities/teamsanduniversities/teamsanduniversities.js:36]: This seems dangerous given this is public source code

Comment (vsoler27) [reply src/pages/teamsanduniversities/teamsanduniversities/teamsanduniversities.js:36]: yeah, im unsure abt that but just to get the functionality working i need a token, trying to figure out what to do bc the endpoints need a token and non logged in users should be able to see all the teams. temporary solution until we figure out something better

**Comment by abp6318** [comment]
Why do the general get requests need a token? Feels kinda counterintuitive to me

File: src/pages/teamsanduniversities/teamsanduniversities/teamsanduniversities.js (hunk 1 above)
Comment (abp6318) [reply src/pages/teamsanduniversities/teamsanduniversities/teamsanduniversities.js:36]: could we just make the requests on the backend not require a token? idk why they need one to begin with

**Comment by vsoler27** [comment]
> Why do the general get requests need a token? Feels kinda counterintuitive to me

yeah <3 thats what i thought too but aidan said we need it, trying to see if i can get him to change it for me

File: src/pages/teamsanduniversities/teamsanduniversities/teamsanduniversities.js (hunk 1 above)
Comment (vsoler27) [reply src/pages/teamsanduniversities/teamsanduniversities/teamsanduniversities.js:36]: let me poke around

**Review by abp6318 — APPROVED** [review]
let's goooooooo

**Pull request merged**